Shared/Samples/Query table statistics/QueryTableStatisticsView.swift

```diff
@@ -0,0 +1,217 @@
+                    }
+                    .labelStyle(.iconOnly)
+                    .popover(isPresented: $isShowingSettings) {
+                        Form {
+                            Toggle("Include Cities Outside Extent", isOn: $includesCitiesOutsideExtent)
+                            Toggle("Include Cities Under 5M", isOn: $includesCitiesUnder5M)
+                        }
+                        .frame(idealWidth: 320, idealHeight: 160)
+                        .presentationCompactAdaptation(.popover)
+                    }
+                }
+            }
+            .errorAlert(presentingError: $queryError)
+    }
+}
+
+/// A view that displays the statistics from a given list of statistic records.
+private struct StatisticList: View {
+    /// The statistic records containing the statistics to display.
+    let records: [StatisticRecord]
+    
+    /// The action to dismiss the view.
+    @Environment(\.dismiss) private var dismiss
+    
+    var body: some View {
+        NavigationStack {
+            List(records, id: \.objectID) { record in
+                Section {
+                    let statistics = record.statistics.sorted(by: { $0.key < $1.key })
+                    ForEach(statistics, id: \.key) { name, value in
+                        LabeledContent(name, value: "\(value)")
+                            .textSelection(.enabled)
+                    }
+                }
+            }
+            .navigationTitle("Statistics")
+            .navigationBarTitleDisplayMode(.inline)
+            .toolbar {
+                ToolbarItem(placement: .confirmationAction) {
+                    Button("Done") {
+                        dismiss()
+                    }
+                }
+            }
+        }
+    }
+}
+
+/// The view model for this sample.
+@Observable
+private final class Model {
+    /// A map with a streets basemap.
+    let map: Map = {
+        let map = Map(basemapStyle: .arcGISStreets)
+        
+        let envelope = Envelope(
+            xRange: -14_803_800...5_233_700,
+            yRange: -15_950_300...17_262_800,
+            spatialReference: .webMercator
+        )
+        map.initialViewpoint = Viewpoint(boundingGeometry: envelope)
+        
+        return map
+    }()
+    
+    /// The service feature table containing the data to query.
+    private let serviceFeatureTable = ServiceFeatureTable(url: .sampleWorldCitiesLayer)
+    
+    /// The parameters to use in the statistics query.
+    private let statisticsQueryParameters: StatisticsQueryParameters = {
+        // Creates a statistic definition for the "POP" field and each of the statistic types.
+        let statisticDefinitions = StatisticDefinition.StatisticType.allCases.map { statisticType in
+            StatisticDefinition(fieldName: "POP", statisticType: statisticType)
+        }
+        
+        return StatisticsQueryParameters(statisticDefinitions: statisticDefinitions)
+    }()
```

Comment:
You could simplify this by not using an immediately evaluated closure.

```suggestion
    private let statisticsQueryParameters = StatisticsQueryParameters(
        // Creates a statistic definition for the "POP" field and each of the statistic types.
        statisticDefinitions: StatisticDefinition.StatisticType.allCases
            .map { .init(fieldName: "POP", statisticType: $0) }
    )
```